components/brave_vpn/pref_names.cc

```diff
@@ -23,6 +24,12 @@ void RegisterProfilePrefs(PrefRegistrySimple* registry) {
                                skus::GetDefaultEnvironment());
 }
 
+void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
```

Comment:
typically you only keep names in `whatever_names.h`. Registering is mostly done by the code that uses the prefs, e.g. the service or something

Reply:
agreed, perhaps we should move both Register methods in another PR if VPN devs agree. thoughts @deeppandya @simonhong ?

Reply:
Ok, I'll f/u this soon.